static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -127,20 +161,58 @@ define([
         }
     }
 
+    function submitDeliveryPreference(event) {
+        event.preventDefault();
+
+        var prefAnswerDeliveryBtn = event.currentTarget;
+        var prefAnswerDelivery = prefAnswerDeliveryBtn.dataset.deliveryMethod;
+        var questionId = prefAnswerDeliveryBtn.dataset.questionId;
```

Comment:
this attribute doesn't seem to be available, can it be removed?

Reply:
So it's not available and has been removed as it was left over from earlier logic. However this now means there is no questionId available to send through in the Ophan event recording. Should we send atomId twice?
`sendNewStyleInteraction(atomId.trim(), 'SUBSCRIBE', prefAnswerDelivery.value, questionId);`

Reply:
The `eventId` (the final parameter) is actually optional in `sendNewStyleInteraction`, so you can just leave it out. It's also optional in the [ophan model](https://github.com/guardian/ophan/blob/cb8bdbb3b75247cf4c74f2b5c679e58915c9e17e/event-model/src/main/thrift/componentevent.thrift#L164)

Reply:
Perfect, thanks Tom!